### PR TITLE
Admin voting times, but in local time

### PR DESCRIPTION
**Pull request by tygor**
At ICCM America 2023, Tom asked me to tackle a feature where the times entered in the admin page for voting could be entered in the desired timezone, but then saved as UTC. This pull request tackles this.
Admin.html now has JavaScript to convert the incoming times to the selected timezone. Admin.php now has logic to convert the posted date and time back to UTC before saving to the DB.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "birds-of-a-feather",
   "private": true,
   "dependencies": {
-    "cypress": "^5.0.0",
+    "cypress": "^4.2.0",
```
Comment (franc6) [inline package.json:5]: Is the downgrade of some of these packages intentional?

File: src/.phpunit.result.cache [hunk 2]
```diff
@@ -0,0 +1 @@
...r::showProjectorViewReturnsNoVoteAndLeaderForNominatingStage":0.001,"TestProjector::showProjectorViewShowsFinishedStageForFinished":0.002,"TestProjector::showProjectorViewShowsVotingStageForLocked":0.002,"TestProjector::showProjectorViewShowsVotingStageForVoting":0.001,"TestResults::calculateResultsWithNoPrepBoF":0.358,"TestResults::calculateResultsWithPrepBofAt01":0.27,"TestResults::calculateResultsWhenNoConflictsEnoughWorkshops34":0.269,"TestResults::calculateResultsConflictsWhenConflictsEnoughWorkshops34":0.274,"TestResults::calculateResultsConflictsWhenConflictsEnoughWorkshops25":0.191,"TestResults::calculateResultsConflictsWhenConflictsEnoughWorkshops52":0.207,"TestResults::calculateResultsConflictsWhenConflictsNotEnoughWorkshops34":0.237,"TestResults::calculateResultsWithMaxConflicts":0.277,"TestResults::calculateResultsThrowsExceptionIfInvalidLocations":0.002,"TestResults::calculateResultsThrowsExceptionIfInvalidRounds":0.001,"TestResults::calculateResultsThrowsExceptionIfTooFewLocations":0,"TestTimezones::testTimezonesList":0.013,"TestTimezones::testTimezonesDateTimeSetTimezone":0,"TestTimezones::testTimezonesParseDateTimeWithTimezone":0,"TestTimezones::testParseAndUtc":0}}
```
Comment (franc6) [inline src/.phpunit.result.cache:1]: We probably want to exclude cache files like this...

**Review by franc6 — CHANGES_REQUESTED** [review]
Sorry it took me so long to look at this!

File: package.json (hunk 1 above)
Comment (tygor) [reply package.json:5]: Nope. I just struggled to get it working. I could test using version 5.0, perhaps tonight.

File: src/.phpunit.result.cache (hunk 2 above)
Comment (tygor) [reply src/.phpunit.result.cache:1]: Sure thing. I can update the .gitignore to accommodate.

**Comment by franc6** [comment]
Closing in favor of an updated PR, since the codebase had been updated without this PR. :(. Thanks again, @tygor, because it's still all your code!

**Pull request closed without merging**